### DISCUSSION TITLE
월드 만들기에 파일 등록시 에러 #120

### DISCUSSION THREAD

**Discussion by nice1378** [naverz/zepeto-studio-kor#198] [4. General]
패키지 파일을 등록할 때 시스템 업로드 실패가 뜹니다.

무엇이 문제인지 알 수 있는 방법이 있나요?

용량은 300M 입니다.

![KakaoTalk_20220111_151841735](https://user-images.githubusercontent.com/5405105/148897849-850bd820-4547-4055-b0bd-f9fe0c136425.png)

**Comment by hyeonjin-jo** [comment]
안녕하세요.

일시적인 서버 문제로 예상됩니다.

불편을 드려서 죄송합니다 현재도 재현이 된다면 답변 부탁드립니다.
